runner/src/server/plugins/router.ts

```diff
@@ -40,13 +40,29 @@ export default {
         {
           method: "get",
           path: "/help/cookies",
-          handler: async (_request: HapiRequest, h: HapiResponseToolkit) => {
-            return h.view("help/cookies");
+          handler: async (request: HapiRequest, h: HapiResponseToolkit) => {
+            const cookiesPolicy = request.state.cookies_policy;
+            let analytics: string | undefined;
+            if (cookiesPolicy) {
+              analytics =
+                cookiesPolicy?.analytics === "on" ? "accept" : "reject";
+            }
```

Comment:
This can just be
```suggestion
            const analytics = cookiesPolicy?.analytics === "on" ? "accept" : "reject";
```

if cookiesPolicy is undefined, then `cookiesPolicy?.analytics` is evaluated as undefined, which is not `=== "on"`.